samples/lightweight-component/.ipynb_checkpoints/lightweight_component-checkpoint.ipynb

```diff
@@ -0,0 +1,359 @@
+{
```

Comment:
We don't need notebook checkpoint for GitHub, can you remove this file?

Reply:
sure